app/src/main/java/nu/yona/app/api/manager/network/RestApi.java

```diff
@@ -328,6 +328,21 @@
     @DELETE
     Call<Void> deleteBuddy(@Url String url, @Header(NetworkConstant.YONA_PASSWORD) String password, @Header(NetworkConstant.ACCEPT_LAUNGUAGE) String acceptLanaguage);
 
+
+
+    /********
+     * NOTIFICATION MANAGER
+     *
+     * @param nextUrl      the url
+     * @param password the password
+     * @return the messages
+     */
+    @GET
+    Call<YonaMessages> getNextSetOfMessages(@Url String nextUrl, @Header(NetworkConstant.YONA_PASSWORD) String password,
```

Comment:
This simply gets the messages from the URL, whether the first page or the next isn't known here, so rename into ``getMessages``.